system/modules/comments/classes/Comments.php

```diff
@@ -269,6 +276,11 @@ protected function renderCommentForm(\FrontendTemplate $objTemplate, \stdClass $
 			}
 
 			$arrField['eval']['required'] = $arrField['eval']['mandatory'];
+
+			/**
+			 * @var \Widget $strClass
+			 * @var \Widget $objWidget
```

Comment:
Is this necessary? Does PhpStorm not notice that `new Widget` is of type widget?